.github/workflows/ci.yml

```diff
@@ -0,0 +1,247 @@
+name: ci
+on:
+  pull_request:
+  push:
```

Comment:
Should this be running when release branches are pushed? As-is, it'll only run on pull requests against the release branch

Reply:
Agreed, should run for release branches. They are practically at the same level of stability as `main`.

Reply:
Adding the release branches